dss_datamover/config/config.json

```diff
@@ -17,6 +17,12 @@
     "port_status": 6001
   },
   "ip_address_family" : "IPV6",
+  "socket_options": {
+    "connection_delay_interval": 2,
+    "connection_time_threshold": 300,
+    "message_length": 10,
```

Comment:
this is confusing for the end users. Change it to response header length as you mentioned in the code.
Also lot of these fields are not understandable for end user. Need to have some simple names or documentation for that.